package.json

```diff
@@ -163,17 +166,20 @@
 		"enzyme": "3.9.0",
 		"enzyme-adapter-react-16": "1.11.2",
 		"enzyme-to-json": "3.3.5",
+		"expect-puppeteer": "4.1.1",
 		"glob": "7.1.3",
 		"gulp-delete-lines": "0.0.7",
 		"husky": "2.2.0",
 		"jest": "24.7.1",
+		"jest-puppeteer": "4.1.1",
 		"lodash": "4.17.11",
 		"mini-css-extract-plugin": "0.5.0",
 		"mocha": "6.1.3",
 		"mockery": "2.1.0",
 		"nock": "10.0.6",
 		"node-wp-i18n": "1.2.3",
 		"prettier": "https://github.com/Automattic/wp-prettier/releases/download/wp-1.17.0/wp-prettier-1.17.0.tgz",
+		"puppeteer": "1.15.0",
```

Comment:
Noting that this is a bit heavier dependency (discussed p1554884363738500-slack-jetpack-gutenberg).

```bash
$ yarn why puppeteer
...
info Disk size without dependencies: "196.74MB"
info Disk size with unique dependencies: "197.57MB"
info Disk size with transitive dependencies: "199.36MB"
```

I don't think it's soooo heavy that it's worth the hassle to make pulling it optional (https://docs.npmjs.com/files/package.json#optionaldependencies, https://docs.npmjs.com/files/package.json#peerdependencies) but just wanted to give heads up.